static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -1,359 +0,0 @@
-import 'intersection-observer'; // polyfill
-
-import {useCallback, useContext, useEffect, useState} from 'react';
-import styled from '@emotion/styled';
-import type {Node} from '@react-types/shared';
-import {motion} from 'framer-motion';
-
-import {addSuccessMessage} from 'sentry/actionCreators/indicator';
-import {
-  DraggableTabList,
-  TEMPORARY_TAB_KEY,
-} from 'sentry/components/draggableTabs/draggableTabList';
-import type {DraggableTabListItemProps} from 'sentry/components/draggableTabs/item';
-import type {MenuItemProps} from 'sentry/components/dropdownMenu';
-import {t} from 'sentry/locale';
-import type {InjectedRouter} from 'sentry/types/legacyReactRouter';
-import normalizeUrl from 'sentry/utils/url/normalizeUrl';
-import {useHotkeys} from 'sentry/utils/useHotkeys';
-import {useLocation} from 'sentry/utils/useLocation';
-import useOrganization from 'sentry/utils/useOrganization';
-import {DraggableTabMenuButton} from 'sentry/views/issueList/groupSearchViewTabs/draggableTabMenuButton';
-import EditableTabTitle from 'sentry/views/issueList/groupSearchViewTabs/editableTabTitle';
-import {
-  type IssueView,
-  IssueViewsContext,
-} from 'sentry/views/issueList/groupSearchViewTabs/issueViews';
-import {IssueSortOptions} from 'sentry/views/issueList/utils';
-import {NewTabContext, type NewView} from 'sentry/views/issueList/utils/newTabContext';
-
-export interface DraggableTabBarProps {
-  initialTabKey: string;
-  router: InjectedRouter;
-}
-
-export const generateTempViewId = () => `_${Math.random().toString().substring(2, 7)}`;
-
-export function DraggableTabBar({initialTabKey, router}: DraggableTabBarProps) {
-  // TODO: Extract this to a separate component encompassing Tab.Item in the future
-  const [editingTabKey, setEditingTabKey] = useState<string | null>(null);
-
-  const organization = useOrganization();
-  const location = useLocation();
-
-  const {cursor: _cursor, page: _page, ...queryParams} = router?.location?.query ?? {};
-  const {viewId} = queryParams;
-
-  const {setNewViewActive, setOnNewViewsSaved} = useContext(NewTabContext);
-  const {tabListState, state, dispatch} = useContext(IssueViewsContext);
-  const {views: tabs, tempView: tempTab} = state;
-
-  useHotkeys(
-    [
-      {
-        match: ['command+s', 'ctrl+s'],
-        includeInputs: true,
-        callback: () => {
-          if (tabs.find(tab => tab.key === tabListState?.selectedKey)?.unsavedChanges) {
-            dispatch({type: 'SAVE_CHANGES', syncViews: true});
-            addSuccessMessage(t('Changes saved to view'));
-          }
-        },
-      },
-    ],
-    [dispatch, tabListState?.selectedKey, tabs]
-  );
```

Comment:
Moved to `customViewsHeader.tsx` 

This logic controls the ctrl/cmd+s to save behavior